src/common/tests/umem_test_bmem.c

```diff
@@ -73,7 +73,7 @@ struct umem_store_ops _store_ops = {
 	.so_wal_submit = _persist_submit,
 };
 
-struct umem_store ustore = { .stor_ops = &_store_ops };
+struct umem_store ustore = { .stor_size = POOL_SIZE, .stor_ops = &_store_ops };
```

Comment:
The correct stor_size is "POOL_SIZE - 4k", won't it cause bmem unit test failure by specifying slightly larger size?

Reply:
The current fix works as long as stor_size <= sz condition is true.